src/Swift.Bindings/src/Demangler/Swift5Reducer.cs

```diff
@@ -83,7 +91,21 @@ IReduction ConvertProtocolWitnessTable (Node node, string? name)
         var impNamed = (NamedTypeSpec)implementingType.TypeSpec;
         var protoNamed = (NamedTypeSpec)protocolType.TypeSpec;
 
-        return new ProtocolWitnessTableReduction() { Symbol = mangledName, ImplementingType = impNamed, ProtocolType = protoNamed };
+        if (isDescriptor) {
+            grandchild = Convert (child.Children [2]);
```

Comment:
Is this safe? Specifically `child.Children [2]`.

Reply:
Guaranteed. If it's a descriptor, that has to be there.